package-lock.json

```diff
@@ -4736,7 +4736,7 @@
       "integrity": "sha512-37RDzLiqSHJjopFg3T/Sqntx793OMQ4o7tkf3IuKe68Mshz3mwavJhIaIX/mca+0u/aWquySJFzlZ4D8EdtRSA=="
     },
     "language-css": {
-      "version": "https://www.atom.io/api/packages/language-css/versions/0.44.4/tarball",
+      "version": "https://www.atom.io/api/packages/language-css/versions/0.44.5/tarball",
       "integrity": "sha512-BBlnxUx9SCv+mIYOBWUApvyB/rSOewy2m8g6f3Vk2LfLf0+t8+Hcay5hWqXqpyKS4BUa15E3TVhl3PHEpqwZ1w==",
```

Comment:
With the version bump for this package, I'd expect the SHA512sum in the "integrity" field to be updated as well.

You can run `npm install --package-lock-only` in the root of the atom repo to update the SHA512sum/integrity field.